cmd/ocid/main.go

```diff
@@ -258,6 +264,16 @@ func main() {
 	}
 
 	app.Action = func(c *cli.Context) error {
+		cp := c.GlobalString("cpu-profile")
+		if c.GlobalBool("debug") && cp != "" {
```

Comment:
So why would we be linking profiling and debugging ? In particular the CPU profiling data you're going to get is going to be tampered by the fairly large amount of debugging traces that ocid generates.
So I'd simply check for `cpu-profile` here.